### DISCUSSION TITLE
How to use this with something like Glitch.com or replit

### DISCUSSION THREAD

**Discussion by tiagozip** [pocketbase/pocketbase#723] [Q&A] [answered]
Hi! Pocketbase seems quite cool to me, and I wanted to test it with Glitch.com or Repl.it, although this doesn't seem easy.
How am I supposed to do this?
Thanks!
Tiago

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not familiar with glitch, but I've tested replit in the past and I don't recommend it, see https://github.com/pocketbase/pocketbase/discussions/353. If you still for some reason prefer replit, you can create a Go file with content similar to the [example in the readme](https://github.com/pocketbase/pocketbase#example) and configure the replit environment to run `go run main.go serve` (or you can also just upload the prebuilt linux executable to your replit project and change your startup command ot `./pocketbase serve`).

If you are looking for free hosting, I recommend checking fly.io - https://github.com/pocketbase/pocketbase/discussions/537.

**Reply by tiagozip** [reply]
Thanks for the answer!
